ZFSin/zfs/module/zfs/zfs_vnops_windows.c

```diff
@@ -1549,10 +1550,11 @@ NTSTATUS pnp_query_id(PDEVICE_OBJECT DeviceObject, PIRP Irp, PIO_STACK_LOCATION
 
 	zmo = (mount_t *)DeviceObject->DeviceExtension;
 
-	Irp->IoStatus.Information = (void *)ExAllocatePoolWithTag(PagedPool, zmo->bus_name.Length + sizeof(UNICODE_NULL), '!OIZ');
-	if (Irp->IoStatus.Information == NULL) return STATUS_NO_MEMORY;
+	Irp->IoStatus.Information = (ULONG_PTR) ExAllocatePoolWithTag(PagedPool, 2 * (zmo->bus_name.Length + 1), '!OIZ');
```

Comment:
the unicode string length field is already in bytes.  you cannot multiply it again.  remove the 2 * here and below.

Reply:
I see. Thanks for catching it. For some reason I thought the length was in wchars.